packages/deployment/app/cloudformation.template.yml

```diff
@@ -981,14 +890,48 @@ Resources:
       KeyName: {{ ecs.keyPairName }}
     {{/if}}
       UserData:
-        "Fn::Base64":
-          Fn::Join:
-            - ""
-            - - "#cloud-boothook\n"
-              - "\n"
-              - "yum install -y aws-cfn-bootstrap\n"
-              - Fn::Sub: "/opt/aws/bin/cfn-init -v --stack ${AWS::StackName} --resource CumulusContainerInstanceLaunch --region ${AWS::Region} -c MountConfig\n"
-              - Fn::Sub: "/opt/aws/bin/cfn-signal -e $? --stack ${AWS::StackName} --resource CumulusECSAutoScalingGroup --region ${AWS::Region}"
+        Fn::Base64:
+          Fn::Sub: |
```

Comment:
I don't see any reason this shouldn't work in CW, presuming the test plan from #901 and the kes CF merge integration test pass.